features/admin/user.feature

```diff
@@ -104,6 +104,7 @@ Feature: User
       | alpha |
       | beta  |
       | gamma |
+    And I wait for the AJAX requests to finish
```

Comment:
Some plugin causes the xhr membership request to become slower than on core, breaking that cuke